### PR TITLE
[Feature] App Store CLI - Make it super easy to add an app.

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Updated Demo with creation/deletion/editing of app - https://www.loom.com/share/b850dafcfe454346b7cc33764d031faa

App Store Cli - Features
- Create an App `yarn app-store create`
- Quickly Edit basic details using cli `yarn app-store edit --slug`
- Delete the app[Cleans up DB as well] `yarn app-store delete --slug`

See roadmap in app-store-cli/README.md

Separated "unification of appId and type" from this PR to avoid complexity. Work going on for that in https://github.com/calcom/cal.com/pull/2953

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by PeerRich** [comment]
amazing cant wait for this to go live! great job. will make generating new apps a blast!

File: packages/app-store-cli/readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Someone can add wrong directory name(which doesn't satisfy slug requirements) manually. How to handle it.
+- Allow editing and updating app from the cal app itself - including assets uploading when developing locally.
+- Improvements in shared code across app
+  - Use baseApp/api/add.ts for all apps with configuration of credentials creation and redirection URL.
+- Delete creation side effects if App creation fails - Might make debugging difficult
+  - This is so that web app doesn't break because of additional app folders or faulty db-seed
+  
+
+Go Live
+- We should fix slack and giphy apps first and ensure that there is no case where appId is null
```
Comment (zomars) [inline packages/app-store-cli/readme.md:43]: Is this still relevant?

**Review by zomars — APPROVED** [review]
My only concern if that this may become quite challenging to maintain and also for newcomers. But this is a step in the right direction. Awesome work as always @hariombalhara  🙏🏽

**Comment by PeerRich** [comment]
super cool pr 👏👏

**Pull request merged**